react/graphql/getContactData.graphql

```diff
@@ -0,0 +1,20 @@
+query getDocuments(
+  $acronym: String
+  $fields: [String]
+  $schema: String
+  $page: Int
+  $pageSize: Int
+) {
+  documents(
+    acronym: $acronym
+    fields: $fields
+    schema: $schema
+    page: $page
+    pageSize: $pageSize
+  )  {
```

Comment:
@context(provider: "vtex.store-graphql") : Not added it here because I was getting 403 error. So, In final commit , I will push it..